src/app/loans/common-resolvers/collateral-template.resolver.ts

```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
```

Comment:
Add the particular condition in the `loan-action-button.resolver.ts` file.

Reply:
Collaterals template parameters are different. Can I use different method for getting collateral template?

Reply:
Yaa sure.